dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilder.java

```diff
@@ -59,8 +60,9 @@ public ClassPathBuilder(DependencyGraphBuilder dependencyGraphBuilder) {
    * closest' strategy follows Maven's dependency mediation.
    *
    * @param artifacts Maven artifacts to check. They are treated as the root of the dependency tree.
+   * @throws IOException if a jar file can't be read
    */
-  public ClassPathResult resolve(List<Artifact> artifacts) {
+  public ClassPathResult resolve(List<Artifact> artifacts) throws IOException {
```

Comment:
ClassPathBuilder should focus on building class path. It should not read the content of JAR.

Reply:
fixed